### DISCUSSION TITLE
Zeus cli cannot build or preview apps on Amazfit Band 7

### DISCUSSION THREAD

**Discussion by fhfuih** [zepp-health/zeppos-forum#105] [Q&A] [answered]
The document says Amazfit band 7 is running ZeppOS 1.0. But after I create a sample app with zeus cli and try to `build` or `preview` it, the CLI won't offer band 7 as an option. The cli version is 1.2.2 (should be the latest). Does anyone know how can I develop apps on Band 7?

**Comment by zdunex25** [comment, marked as answer]
You have to add correct target manually: https://docs.zepp.com/docs/reference/app-json/#targets-object
based on this table https://docs.zepp.com/docs/reference/related-resources/device-list/#devices-with-zepp-os

**Comment by wviana** [comment]
Found [this example](https://github.com/fwz233/Anynews/blob/main/app.json) that makes it generic for most devices. I was able to run it in my Band 7.

Hope it help others.
